sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -157,3 +159,16 @@
     ('sqlserver.database.master_files.size', 'sys.master_files', 'size'),
     ('sqlserver.database.master_files.state', 'sys.master_files', 'state'),
 ]
+
+DATABASE_FILES_IO = [
+    ('sqlserver.files.reads', 'num_of_reads', AgentCheck.monotonic_count),
+    ('sqlserver.files.read_bytes', 'num_of_bytes_read', AgentCheck.monotonic_count),
+    ('sqlserver.files.read_io_stall', 'io_stall_read_ms', AgentCheck.monotonic_count),
+    ('sqlserver.files.read_io_stall_queued', 'io_stall_queued_read_ms', AgentCheck.monotonic_count),
+    ('sqlserver.files.writes', 'num_of_writes', AgentCheck.monotonic_count),
+    ('sqlserver.files.written_bytes', 'num_of_bytes_written', AgentCheck.monotonic_count),
+    ('sqlserver.files.write_io_stall', 'io_stall_write_ms', AgentCheck.monotonic_count),
```

Comment:
should this be a gauge instead of a count?

Reply:
oh these are all just counters huh

> The counters are initialized to empty whenever the SQL Server (MSSQLSERVER) service is started.

Reply:
Count is correct IMO. Because these counters are monotonically increasing and tracked on the database side, we are able to capture 100% of hits by taking the deltas between check runs.